packages/plugin/src/lib/paystack.ts

```diff
@@ -1,4 +1,5 @@
-import https from "https";
+// import https from "https";
+import axios from "axios";
```

Comment:
You need to add axios to the dependencies. `pnpm add axios` in `/packages/plugin`